src/App.js

```diff
@@ -1,12 +1,16 @@
-import React from 'react';
+import React, { useState } from 'react';
```

Comment:
De useState wordt niet meer gebruikt binnen de file
```suggestion
import React from 'react';
```